test/entities/swapV3.test.ts

```diff
@@ -76,14 +77,17 @@ describe('SwapV3', () => {
                             {
                                 pool: path3hops.pools[0],
                                 tokenOut: path3hops.tokens[1].address,
+                                isBuffer: false,
                             },
                             {
                                 pool: path3hops.pools[1],
                                 tokenOut: path3hops.tokens[2].address,
+                                isBuffer: false,
                             },
                             {
                                 pool: path3hops.pools[2],
                                 tokenOut: path3hops.tokens[3].address,
+                                isBuffer: false,
```

Comment:
I don't like veering away from pattern of declaring all path hopping setup in const vars at start of test, but I wasn't sure if its okay or makes sense to add `isBuffer` to the [Path type](https://github.com/balancer/b-sdk/blob/main/src/entities/swap/paths/types.ts#L6)

Also do we need to account for situations where some but not all of the hops will require `isBuffer: true` ?

Reply:
For now this is fine. We'll be revisiting buffer pools later.